### ISSUE TITLE
Support for defining custom cron jobs

### ISSUE THREAD

**Issue by jabbors** [giner/ansible-role-barman#3]
I have a use case where I'd like to manage custom cron jobs for each backup job, ie I'd want to backup the backups to another remote server using tools like [borg](https://www.borgbackup.org) or [restic](https://restic.net).

The configuration could look something like

```
- name: my-server
    params:
      - <irrelevant>
    pgpass:
      - <irrelevant>
    backup_schedule:
      cron:
        hour: 2
        minute: 11
      custom:
        job: 'borg create ssh://user@remote.server.com:/home/barman/my-server::$(date --iso-8601=date) /var/lib/barman/my-server'
        cron:
          hour: 3
          minute: 13
```

Do you think this would be useful? I can create a PR if you think it will be valuable for this module.

**Comment by jabbors** [comment]
The custom section can even be a list. I'll wrap up a PR for this.

**Comment by giner** [comment]
Implemented with #6
